strongbox-resources/strongbox-common-resources/src/main/resources/logback-spring.xml

```diff
@@ -92,14 +92,6 @@
 
     <if condition="${debug:-false}">
     <then>
-        <!-- ApacheDS: Used for testing -->
```

Comment:
AFAIK we still using ApacheDS in tests.

Reply:
or not? :)
what is `unboundid-ldapsdk` ?

Reply:
The development of ApacheDS has become stangnant. The newer versions are incompatible with recent versions of Spring Boot/Security. UnboundID is an alternative Java LDAP server, that is perhaps the most sophisticated LDAP implementation in Java. Here are some details on the licensing: https://ldap.com/unboundid-ldap-sdk-for-java/.

It doesn't have dependency incompatibilities and works out of the box. We were previously using this already via the `unboundid-maven-plugin` that I had created.